cmd/minishift/cmd/start.go

```diff
@@ -411,8 +412,14 @@ func exportContainerImages(hostVm *host.Host, version string) {
 
 // calculateDiskSizeInMB converts a human specified disk size like "1000MB" or "1GB" and converts it into Megabits
 func calculateDiskSizeInMB(humanReadableDiskSize string) int {
+	// pre-process human readable string
+	if stringUtils.HasOnlyNumbers(humanReadableDiskSize) {
+		humanReadableDiskSize += "M"
+	}
+
 	diskSize, err := units.FromHumanSize(humanReadableDiskSize)
 	if err != nil {
+		fmt.Println("")
```

Comment:
Can we please add some tests for this function? This will also document on what and what not is acceptable as input via the --memory flag.